src/presentation/http/router/note.ts

```diff
@@ -26,10 +26,25 @@ const NoteRouter: FastifyPluginCallback<NoteRouterOptions> = (fastify, opts, don
   const noteService = opts.noteService;
 
   /**
-   * Get notes
+   * Get note by id
    */
-  fastify.get('/', async () => {
-    return { hello: 'world' };
+  fastify.get<{ Params: GetNoteByIdOptions }>('/:id', async (request, reply) => {
+    const params = request.params;
+
+    const note = await noteService.getNoteById(params);
```

Comment:
do not pass params directly to the domain. Extract and validate data first.

Reply:
done